### PR TITLE
refactor(rxCheckbox): migrate to Elements > Forms

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-275

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

_screenshots to follow_

### CODE REVIEW

File: demo/templates/rx-example.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="example">
+<div class="rx-example">
```
Comment (CITguy) [inline demo/templates/rx-example.html]: Same change as in #1646

**Comment by CITguy** [comment]
### Nav Menus
before | after
----- | -----
screen shot 2016-05-05 at 9 04 51 pm | screen shot 2016-05-05 at 9 05 06 pm

### Page Content

#### Before
local-elements-forms-before

#### After

* No more Atoms!
* Corrected rxExample styles (same as #1646)
* rxCheckbox content now at Elements > Forms ("Checkboxes")

local-elements-forms-after

**Comment by glynnis** [comment]
Design LGTM

**Comment by parlarjb** [comment]
LGTM

**Comment by halkon** [comment]
LGTM unless you really wanna do vidyo.

**Pull request merged**